tests/test-lib/sleep.ts

```diff
@@ -0,0 +1,3 @@
+export default function sleep(ms: number) {
+	return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
```suggestion
import { promisify } from 'util';
export default = promisify(setTimeout);
```

Reply:
updated. nice had not seen/thought of using `promisify` with `setTimeout` before